var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -61,19 +61,31 @@ def autoreconf(self, spec, prefix):
         aclocal('-I', 'lib/autoconf')
         autoconf()
 
+    @property
+    def optflags(self):
+        flags = '-O2'
+
+        # Optimizations for the Intel compiler, suggested by CP2K
+        if '%intel' in self.spec:
+            # -xSSE2 will make it usable on old architecture
+            flags += ' -xSSE2 -xAVX -axCORE-AVX2 -ipo'
+
+        return flags
+
+    def setup_environment(self, build_env, run_env):
```

Comment:
Can you change `build_env` to `spack_env` to match every other usage in the documentation? Or change `spack_env` to `build_env` everywhere else. I kind of prefer `build_env` now that I think about it.